src/module3/provided/task-3-export-csv-to-txt.ts

```diff
@@ -0,0 +1,81 @@
+import * as fs from 'fs';
+import { pipeline, Transform } from 'stream';
+import readline from 'readline';
+
+/**
+ * Exports a CSV file to a TXT file.
+ * CSV format:
+ *  Book;Author;Amount;Price
+ *  The Compound Effect;Darren Hardy;5;9,48
+ * TXT format: {"book":"The Compound Effect","author":"Darren Hardy","price":9.48}
+ * @param {string} csvPath - The path to the source CSV file.
+ * @param {string} txtPath - The path to the destination TXT file.
+ * @returns {Promise<boolean>} - A promise that resolves to true when export is done.
+ */
+export const exportCsvToTxt = (csvPath: string, txtPath: string): Promise<boolean> => {
```

Comment:
![module3-alternative](https://github.com/user-attachments/assets/ea0a5e79-ef73-45c3-b1f5-e05e736e2ce2)
Solution with csvtojson and pipeline